imports/plugins/core/core/server/startup/startNodeApp.js

```diff
@@ -41,15 +37,13 @@ export default async function startNodeApp({ onAppInstanceCreated }) {
       async createUser(options) {
         return Accounts.createUser(options);
       },
-      queries: coreQueries,
-      mutations: coreMutations,
       rootUrl: ROOT_URL,
+      mutations: {},
```

Comment:
Why are `mutations` and `queries` passed if they are empty?

Reply:
There's code that requires it and this was easier than refactoring that code. This is temporary and will be resolved when we finish entirely separating this into a Node app.